aws-android-sdk-cognitoauth/src/androidTest/java/com/amazonaws/mobileconnectors/cognitoauth/LocalDataManagerIntegrationTest.java

```diff
@@ -214,7 +214,52 @@ public void onFailure(Exception e) {
                     setFromString("email,profile,openid"));
 
             printSharedPreferencesKeys();
-            verifySharedPreferencesForCachedState(auth.awsKeyValueStore);
+            verifySharedPreferencesForCachedState(auth.awsKeyValueStore, "proofKey", "email,profile,openid");
+        } catch (Exception ex) {
+            ex.printStackTrace();
+            assertTrue("Error occurred" + ex, false);
+        }
+    }
+
+    @Test
+    public void testCognitoAuthCachedStateForNoScopes() {
```

Comment:
can you rename this to `testCognitoAuthCachedStateForEmptyScopes`?

Reply:
The test name is from our code base.